Core/Commands/Definitions/OperatorCommands.cs

```diff
@@ -0,0 +1,58 @@
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading.Tasks;
+using ArgsParsing.Types;
+using Persistence.Models;
+
+namespace Core.Commands.Definitions
+{
+    public class StopToken
+    {
+        public bool ShouldStop { get; set; }
+    }
+
+    public class OperatorCommands : ICommandCollection
+    {
+        private readonly StopToken _stopToken;
+        private readonly ImmutableHashSet<string> _operatorNamesLower;
+
+        public OperatorCommands(
+            StopToken stopToken,
+            IEnumerable<string> operatorNames)
+        {
+            _stopToken = stopToken;
+            _operatorNamesLower = operatorNames.Select(s => s.ToLowerInvariant()).ToImmutableHashSet();
+        }
+
+        public IEnumerable<Command> Commands => new[]
+        {
+            new Command("stop", Stop),
+        }.Select(cmd => cmd.WithCondition(
+            canExecute: ctx => IsOperator(ctx.Message.User),
+            ersatzResult: new CommandResult { Response = "Only operators can use that command" }));
+
+        private bool IsOperator(User user) => _operatorNamesLower.Contains(user.SimpleName);
+
+        private async Task<CommandResult> Stop(CommandContext context)
```

Comment:
when running simultaniously, we should have !stop new/!stop new cancel, or !stop old/!stop old cancel to only stop one of the cores, ofc !stop should stop both if no other args are specified, same with !stop cancel